estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,68 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "estate property offer"
+    _order = 'price desc'
+
+    @api.depends("validity")
+    def _compute_deadline(self):
+        for record in self:
+            if record.create_date:
+                record.deadline = record.create_date.date() + relativedelta(days=record.validity)
+            else:
+                record.deadline = fields.Date.today() + relativedelta(days=record.validity)
```

Comment:
```suggestion
            record.deadline = record.create_date.date() or fields.Date.today() + relativedelta(days=record.validity)
```
if `create_date` will be there that value will be considered else today will be taken into account when we have the same check and assigning value this is a better way